Monopoly/Assets/Scripts/Property.cs

```diff
@@ -75,6 +79,24 @@ public bool IsOwningBy(int PlayerId)
         if (PlayerId == ownerId) return true;
         else return false;
     }
+    public bool IsAbleToBuild()
+    {
+        if (deposit)
+            return false;
+        return ableToBuild;
```

Comment:
Możemy mieć przypadek, że zwracane ableToBuild jest undefined?

Reply:
bool nie może być undefined z tego co wiem

Reply:
Tak, ale chodziło mi o to, że wyżej w kodzie przy deklaracji tej zmiennej nie ustawiamy jej wartości ani na true, ani na false. I teraz czy jesteśmy w stanie dotrzeć do tego miejsca w kodzie nadal bez ustawienia wartości.

Reply:
Oficjalnie: w c# domyślnie bool jest ustawiony na false, więc wszystko jest w porządku